pandas/io/json/_json.py

```diff
@@ -286,6 +286,9 @@ def __init__(
             )
             raise ValueError(msg)
 
+        if obj.index.nlevels == 1 and isinstance(obj.index, MultiIndex):
```

Comment:
can you do this in build_table_schema instead

Reply:
Sure